graphrag/query/structured_search/global_search/map_system_prompt.py

```diff
@@ -4,6 +4,13 @@
 """System prompts for global search."""
 
 MAP_SYSTEM_PROMPT = """
+You should always follow the instructions and output a valid JSON object.
```

Comment:
Hi!
Can we divide this into two separate PRs?
Prompt changes require some additional testing on my end, and I want to make sure we merge LLM config for global and local search.

Reply:
I have already rollback MAP_SYSTEM_PROMPT